tests-with-openai/test_data/evaluation_1.yaml

```diff
@@ -3,7 +3,12 @@
   question_text: List all transformers within Substation OSLO
   reference_answer: OSLO T1, OSLO T2
   reference_steps:
-  - - name: sparql_query
+  - - name: retrieval
+      args:
+        query: transformers Substation OSLO
+        k: 2
```

Comment:
Should we remove this?

Reply:
I think so, as far as I understand we use the actual k, and also we probably don't want to specify what k is in the reference corpus.